app/views/jobs/_form.html.erb

```diff
@@ -1,12 +1,22 @@
-<%= form_for [@company, @job] do |f| %>
-  <%= f.label :title %>
-  <%= f.text_field :title %>
+<div class="buttonForm jobForm">
+  <%= form_for [@company, @job] do |f| %>
+    <%= render partial: "shared/errors", locals: { target: @job } %>
 
-  <%= f.label :description %>
-  <%= f.text_field :description %>
+    <%= f.label :title %>
+    <%= f.text_field :title %><br/>
 
-  <%= f.label :level_of_interest %>
-  <%= f.number_field :level_of_interest %>
+    <%= f.label :description %>
+    <%= f.text_field :description %><br/>
 
-  <%= f.submit %>
-<% end %>
+    <%= f.label :level_of_interest %>
+    <%= f.number_field :level_of_interest %><br/>
+
+    <%= f.label :category %>
+    <%= f.collection_select :category_id, Category.all,:id,:title, include_blank: true %><br/>
```

Comment:
again, this is an example of too much logic in your views. you should pass `Category.all` in as an instance variable from your controller!